src/lib/rest/httpRequestSend.cpp

```diff
@@ -652,6 +666,12 @@ int httpRequestSend
   get_curl_context(_ip, &cc);
   if (cc.curl == NULL)
   {
+    //
+    // FIXME P4: servicePath may contain more than one service path, i.e.  /A,/B.
+    //           Only if coming from a forward of a query (postQueryContext())
+    //           This is taken care of in metricsMgr.add(), but could be moved here instead
+    //           as it is the only place where this can happen (99% sure about this).
+    //
```

Comment:
Discussed at skype, to use the funtion mentioned here https://github.com/telefonicaid/fiware-orion/pull/2785/files#r93621406